_gsocproposals/2019/proposal_AstroLabSparkfits.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Apache Spark: migrating FITS Data Source API to V2
```

Comment:
The colon (`:`) after `Apache Spark` makes Jekyll think that `Apache Spark` is an attribute so it gets confused generating the content. This can be solved by wrapping the whole title in quotes:

```suggestion
title: "Apache Spark: migrating FITS Data Source API to V2"
```